### DISCUSSION TITLE
中央大學的鬼故事

### DISCUSSION THREAD

**Discussion by sjh501** [1101-GS4719-C/Python-Community-GS4719-C#1441] [General]
從這學期開始我跟室友都在宿舍遇到一些很玄的事情，像是房門被敲但打開來外面沒有人，莫名的聽到手機的震動聲但找不到來源，有人被鬼壓床壓了兩次，還有遇到一些奇怪的事情。
上網查幾乎找不到學校的鬼故事，但據一位待在中央很久的教授說中央很常鬧鬼，像是有些地下室有出事，或者以前的學長姐有留下一些禁忌之類的。不曉得大家是否也有一些中央的鬼故事。

**Comment by noratsaiiii** [comment]
沒聽過，但中央很多地方看起來都很恐怖，像是宿舍前面的小河

**Comment by TW-Maverick** [comment]
想知道是哪間宿舍？我也常在宿舍遇到一些恐怖的故事，像是都已經半夜一點了，還有猴子在亂叫，現在想起來還是感覺毛毛的，因為我記得學校沒有猴子啊。在去年開學回學校時，停在宿舍前的腳踏車忽然消失了，我超級害怕，以為是鬼把我的腳踏車拿去當作它的點心，幸好它已經很飽了，最後還是奇蹟地找到了！

**Comment by Alexgns** [comment]
曾經聽過學長說文院那邊的建築物本來要用八卦陣，但蓋反導致反而聚集陰氣，有人下去地下室後就都沒有上來了，但我後來去看那邊好像沒有地下室，不知道是不是亂說的

**Comment by pigeon9975** [comment]
我有聽過十年建設那邊的鬼故事，聽說之前有工人在那邊失足摔落，之後政府就來檢查發現公安不合格，所以才會因此換廠商。蓋好之後，有人說教研大樓的飲水機會滴水，因為是要留給之前不幸身亡的工人喝的。

**Comment by zephyrxin01** [comment]
我朋友表示：文院真的很陰，說文解字解構一下文院的建築設計，方方正正的文院中間一棵樹，不就是困嗎。
我：怪不得每次都覺得那邊很涼。

**Comment by aa103967622** [comment]
這是親身經歷的故事！！！

我住在男十三舍三樓，事情發生在半夜，我那時候剛處理完通識課的期末報告爬上床，記得已經是凌晨三點三十分左右，因為黑漆漆的，我找不太到蚊帳的拉鍊，花了好大一番功夫才上床，途中弄出了點聲響，怕是不小心把隔壁床的室友吵醒，我瞄了一眼確認他還熟睡著，然後，我脫下眼鏡，準備躺下進入甜蜜的夢鄉......

突然！我聽到隔壁床的室友叫了一聲我的名字，我以為是我吵醒他了，所以連忙跟他道歉：「對不起啦吵到你睡覺了。」結果，他還是一直叫著我的名字，我驚覺不對，趕緊把眼鏡戴上，發現我隔壁床的室友上半身坐起，眼睛睜大猛盯著我看，用力得像是要把我看穿了，嘴裏還不停念叨著我的名字⋯⋯

我嚇得躲進被窩裡不敢再看任何一眼，直到天亮，那名室友醒來後完全不記得自己有起來叫過我，然後這件事就這樣不了了之。

事後回想起來還是覺得，當時我的室友一定是被什麼髒東西附身了......

在三樓從二樓上來的樓梯是不是有塊超大鏡子，那東西感覺陰陽怪氣的，不知道有沒有人知道它的來歷？
